portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,7 +26,26 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    // Convert to JSON
+    String message1 = "My name is Micayla.";
+    String message2 = "I am from the Bay Area.";
+    String message3 = "I live in New York City.";
+    List<String> micaylaMessages = new ArrayList<>();
+    micaylaMessages.add(message1);
+    micaylaMessages.add(message2);
+    micaylaMessages.add(message3);
+
+    String json = convertToJsonUsingGson(micaylaMessages);
+
+    //Send response as JSON
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
   }
-}
+
+  private String convertToJsonUsingGson(List<String> messages) {
```

Comment:
Since this method does not require access to any instance-level data (methods or attributes), it can (should) be `static`.